src/promptflow/promptflow/core/_utils.py

```diff
@@ -143,3 +147,92 @@ def init_executable(*, flow_dag: dict = None, flow_path: Path = None, working_di
 
     # for DAG flow, use data to init executable to improve performance
     return ExecutableFlow._from_dict(flow_dag=flow_dag, working_dir=working_dir)
+
+
+# !!! Attention!!!: Please make sure you have contact with PRS team before changing the interface.
```

Comment:
will this break PRS when we move this function?

Reply:
I remember they are using the FlowExecutor.update_environment_variables_with_connections, I'll double confirm about this

Reply:
Just confirmed.

Reply:
mmm.. then we should remove this attention now